### DISCUSSION TITLE
sort integrations on info panel by setup time

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#9622] [Dashboard]
please consider allowing the integrations on the info panel to be sorted by Setup time.
It would make it very easy to spot the top time takers, and allow us to focus on the top 10 % while optimizing the instance

the alphabetical order doesn't really add to the functionality, of course being agnostic to the setup time in the first place

<img width="751" alt="Schermafbeelding 2021-07-27 om 14 49 52" src="https://user-images.githubusercontent.com/33354141/127156384-0edead0e-cd18-47ac-8982-0fe6990e7a73.png">

**Comment by ludeeus** [comment]
alphabetical makes it easier to find a specific integration, if implemented it should IMO be a choice with default to the current

**Reply by Mariusthvdb** [reply]
of course. I was thinking  a method just like in the /config/entities panel clicking the column header
